### ISSUE TITLE
LibGUI: Table view selection/cursor desync after mouseup

### ISSUE THREAD

**Issue by awesomekling** [SerenityOS/serenity#4795]
To reproduce, mousedown on a process in the system monitor process table, then move the mouse, and mouseup over another process.

The view cursor will now be where you mousedown'ed, but the view selection is where you mouseup'ed.

**Comment by matthewbjones** [comment]
@awesomekling are you still able to reproduce this? I followed your description and both the highlight and cursor end up where the mouseup occurs.

**Comment by awesomekling** [comment]
Yes, I can reproduce this with current master @ 94551149d1b5b468f5d8779386178ba9ee9207fe

**Comment by jimmy-loyola** [comment]
I tried to reproduce it but couldn't see the described behavior.
I can observe, the row is selected on mousedown and mouseup.
I note that in Linux systems the selection occurs "only" on mousedown, I think it makes sense.
